.circleci/config.yml

```diff
@@ -44,6 +44,51 @@ jobs:
           working_directory: ./web
           command: yarn test:unit
 
+  test_e2e:
+    docker:
+      - image: circleci/python:3.6-node-browsers
+      - image: viime/opencpu-base
+        command: |
+          /bin/sh -c "chmod a+w /usr/local/lib/R/site-library && service cron start && /usr/lib/rstudio-server/bin/rserver && apachectl -DFOREGROUND"
+    steps:
+      - checkout
+      - restore_cache:
+          name: Restore Yarn Package Cache
+          keys:
+            - yarn-packages-{{ checksum "e2e-test/package.json" }}
+      - save_cache:
+          name: Save Yarn Package Cache
+          key: yarn-packages-{{ checksum "e2e-test/package.json" }}
+          paths:
+            - ~/.cache/yarn
+      - run:
+          working_directory: ./e2e-test
+          name: Install dependencies
+          command: yarn
+      - run:
+          working_directory: ./e2e-test
+          name: Run lint tests
+          command: yarn run lint
+      - run:
+          working_directory: ./e2e-test
+          name: Serve the website
+          command: yarn serve
+          background: true
+      # flask run
+      - run:
+          working_directory: ./e2e-test
+          name: Start the development server
+          command: flask run
+          background: true
```

Comment:
This step is failing because you haven't installed anything python related yet, but are already trying to invoke `flask run`. See [dandiarchive](https://github.com/dandi/dandiarchive/blob/master/.circleci/config.yml#L50-L66) for an example of how to install the app prior to serving it.